sos/plugins/gnocchi.py

```diff
@@ -38,11 +38,17 @@ def setup(self):
 
         self.limit = self.get_option("log_size")
         if self.get_option("all_logs"):
-            self.add_copy_spec("/var/log/gnocchi/",
-                               sizelimit=self.limit)
+            self.add_copy_spec([
+                "/var/log/gnocchi/",
+                "/var/log/containers/gnocchi/"],
+                sizelimit=self.limit
```

Comment:
There's a bug here in the original code (not that you're adding one, but I'd rather we fixed it up as we have the opportunity).

The `limit` parameter does not work effectively when passed a directory (since the size check is applied to the directory inode, which is then passed whole to the archival routines).

Any `add_copy_spec()` call that defines a limit needs to pass a glob that will expand to a set of files (not directories).